src/libs/ajax.js

```diff
@@ -944,8 +944,8 @@ const Workspaces = signal => ({
         return res.json()
       },
 
-      importJob: async (url, filetype) => {
-        const res = await fetchOrchestration(`api/${root}/importJob`, _.mergeAll([authOpts(), jsonBody({ url, filetype }), { signal, method: 'POST' }]))
+      importJob: async (url, filetype, options) => {
```

Comment:
This could use a [default parameter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) to avoid having to pass `null` when there are no options.

```suggestion
      importJob: async (url, filetype, options = null) => {
```